concrete-core/src/prelude.rs

```diff
@@ -1,9 +1,16 @@
 #![doc(hidden)]
 
-pub use super::specification::engines::*;
-pub use super::specification::entities::*;
+// Expose concrete_commons types in the prelude
+// This avoids having to add concrete-commons as a dependency
+// in crates built on top of concrete-core
+pub use concrete_commons::dispersion::*;
+pub use concrete_commons::key_kinds::*;
+pub use concrete_commons::parameters::*;
+pub use concrete_commons::*;
```

Comment:
I know I proposed this, but I think it should be in another commit / PR since its not related to making serde an optional feature

Reply:
Totally right. Maybe I'll just keep it as a separate commit in this PR if this is ok for you?

Reply:
That looks fine to me !